src/PreconditionerTypes.f90

```diff
@@ -201,8 +410,8 @@ SUBROUTINE init_LU_PreCondtype(thisPC,A)
   !   ALLOCATE(PETScMatrixType :: thisPC%U)
   !
   !   !initialize L and U
-  !   SELECTTYPE(U => thisPC%U); TYPE IS(PETScMatrixType)
-  !     SELECTTYPE(L => thisPC%L); TYPE IS(PETScMatrixType)
+  !   SELECT TYPE(U => thisPC%U); TYPE IS(PETScMatrixType)
+  !     SELECT TYPE(L => thisPC%L); TYPE IS(PETScMatrixType)
   !       ! Initialize L and U (add preallocation eventually)
   !       CALL PL%add('MatrixType->matType',SPARSE)
   !       CALL PL%add('MatrixType->n',mat%n)
```

Comment:
Is this block still needed?  Or is it going to be uncommented in the next PR?

Reply:
No, this stuff was here before and I didn't actually change it. It just got caught by a find/replace for SELECT TYPE. It's part of the original LU_PreCondtype.
I'll drop the change to the commented lines

Reply:
Or just deleted the dead code?

Reply:
I can but I didn't write it and don't know for sure the reason it was commented out.

According to git blame it was commented out on 2017-07-18 by Mitchell Young as part of a MatrixTypes refactor.

Reply:
My input would be to delete. I would say after nearly three years, the functionality has either been replaced or is not needed.